lib/fetch.js

```diff
@@ -80,7 +80,7 @@ async function fetch(url, type, date) {
       date = process.env['SCRAPE_DATE'];
     }
     else if (!date) {
-      date = datetime.getYYYYMD();
+      date = datetime.getYYYYMMDD();
```

Comment:
This date is used to retrieve from cache, which uses YYYY-M-D.

Reply:
I'll go ahead and change the cache to use YYYY-MM-DD after this

Reply:
Glad someone took a look at this, it was on my list!